cubeseed/settings.py

```diff
@@ -174,11 +179,31 @@
     ],
 }
 
-# FIXME: this is a simplification for the MVP, should be using cloud storage.
-MEDIA_ROOT = BASE_DIR / "media"
-MEDIA_URL = "/media/"
+# # FIXME: this is a simplification for the MVP, should be using cloud storage.
+# MEDIA_ROOT = BASE_DIR / "media"
+# MEDIA_URL = "/media/"
```

Comment:
you left these commented, can you remove them?

Reply:
Okay, they are now removed.